presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfigurationStaticUpdater.java

```diff
@@ -50,7 +50,7 @@
 import static org.apache.hadoop.hive.conf.HiveConf.ConfVars.COMPRESSRESULT;
 import static org.apache.hadoop.io.SequenceFile.CompressionType.BLOCK;
 
-public class HdfsConfigurationUpdater
+public class HdfsConfigurationStaticUpdater
```

Comment:
Let's name this `HdfsConfigurationInitializer` since it's purpose is to provide the initial configuration.